app/controllers/users_controller.rb

```diff
@@ -4,6 +4,11 @@ def new
 end
 
   def show
+    if User.all == []
+      render file: "/public/404"
+    else
+      @user = User.last
+    end
```

Comment:
@judithpillado wondering if login/logout will change how we do this?
@TravisGM92 created this if you have questions.